src/pages/StartPage.tsx

```diff
@@ -72,7 +81,7 @@ const StartPage = () => {
               primary
               to="/contests/0"
               id="next"
-              aria-label="Select next to start voting."
+              aria-label="Press the right button to advance to the first contest."
```

Comment:
"right button" here vs. "down arrow" and "right arrow" in `ContestPage`. Which do we prefer?

Reply:
Thanks. Updated to use "button" everywhere.